src/fides/core/config/security_settings.py

```diff
@@ -46,40 +46,23 @@ class SecuritySettings(FidesSettings):
     env: SecurityEnv = SecurityEnv.DEV
 
     cors_origins: List[str] = []
```

Comment:
Hey @sanders41, I had to set this back to a `str` to facilitate the `docker run -e FIDES__SECURITY__CORS_ORIGINS="..." fides` use case.